src/components/advanced-color-picker/__snapshots__/advanced-color-picker.spec.js.snap

```diff
@@ -1,7 +1,358 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`AdvancedColorPicker when uncontrolled should render internal composition to match uncontrolled snapshot 1`] = `
-<styled.div>
+<styled.div
```

Comment:
Theme should not be rendered here

in advanced-color-picker.spec.js do that:

`import { noThemeSnapshot } from "../__spec_helper__/enzyme-snapshot-helper"; (edited) 
expect(noThemeSnapshot(wrapper)).toMatchSnapshot();`